tomcat/src/main/java/org/apache/coyote/http11/HttpPath.java

```diff
@@ -0,0 +1,35 @@
+package org.apache.coyote.http11;
+
+import org.apache.commons.lang3.StringUtils;
+
+public class HttpPath {
+
+    private static final String DELIMITER = "\\?";
+    private static final int PATH_INDEX = 0;
+    private static final int QUERY_INDEX = 1;
+
+    private final String path;
+    private final Query query;
+
+    private HttpPath(String path, String queryString) {
```

Comment:
```suggestion
    public HttpPath(String path, Query query) {
```

Query 타입을 가지는 새로운 생성자도 추가해 보면 어떨까?